resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,152 @@
+"""Converting bam to bed and normalised bigwig files."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    Process,
+    SchedulingClass,
+    FileField,
+    StringField
+)
+
+
+class BamToBed(Process):
+    """Takes in a bam file and calculates a normalization factor in .bed format.
+    Done by sorting with samtools and transformed with bedtools.
+    """
+
+    slug = "bam-to-bed"
+    name = "Bam to bed"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Badtools bamtobed ({{alignment|sample_name}})"
+    version = "1.0.0"
+    process_type = "data:bed"
+    category = "Other"
+    entity = {
+        "type": "sample",
+        "input": "alignment",
+    }
+    scheduling_class = SchedulingClass.BATCH
+
+    class Input:
+        """Input fields."""
+
+        alignment = DataField("alignment:bam", label="Alignment BAM file")
+        species = StringField(
+            label="Species",
+            choices=[
```

Comment:
This seems to be a generally useful process. Allowing custom choices (`allow_custom_choice=True`) for species would maybe be useful.